Brokkr/src/main/cpp/commands/SetArmPostitionForDistance.cpp

```diff
@@ -18,57 +27,74 @@ namespace{
     return angle = atan(height/distance);
   }
 
-  double GetNodeDistance(const NodeLevel& NodeLevel, double distance)
+    const double kHybridHeight = 0;
+    const double kMiddleNodeHeight = 34;
```

Comment:
Seems like there are 2 middle heights and 2 upper heights. 

```
    const double kMiddleConeNodeHeight = 34;
    const double kMiddleCubeNodeHeight = 34;
```

Reply:
Is there an assumption that the difference between these heights doesn't matter?